internal/commands/pull/command.go

```diff
@@ -138,6 +166,19 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		ui.Print(terminal.NewDebugLog("Fetched hosting assets"))
 	}
 
+	successfulTemplateWrites := make([]string, 0, len(clientZipPkgs))
+	// TODO(REALMC-9452) fix to iterate in a deterministic order
+	for templateID, templateZipPkg := range clientZipPkgs {
+		if err := local.WriteZip(filepath.Join(pathFrontend, templateID), templateZipPkg); err != nil {
+			return fmt.Errorf("unable to save template '%s' to disk: %s", templateID, err)
+		}
+		// TODO(REALMC-9452): defer printing the successfully saved templates until after the `Successfully pulled app down' log
+		successfulTemplateWrites = append(successfulTemplateWrites, templateID)
+	}
+	if len(successfulTemplateWrites) != 0 {
```

Comment:
is it easy enough to move this down below `"Successfully pulled app down"`?  could probably remove the above `TODO` if so